user/enterprise/ssl-certificate-management.md

```diff
@@ -0,0 +1,268 @@
+- Enter the paths or select the files to upload
+- Click 'Save' at the bottom of the page
+
+**Please note that this will require a restart of Travis CI Enterprise in order to take affect and may cause brief downtime.**
+
+> A valid x509 certificate and private key files are required. The certificate and key must be in PEM format. The key must be unencrypted. 
+
+## Using Intermediate Certificate Chains
+
+In some cases it may be required to provide the [intermediate certificate chain](https://support.dnsimple.com/articles/what-is-ssl-certificate-chain/) for your desired certificate. Reasons for this can vary depending on your internal setup and your root certificate provider.
+
+If you have uploaded a valid certificate but are still seeing unexpected issues related to SSL verification errors then you can set your intermediate cert chain in the settings.
+
+To add your certificate chain:
+
+- Log into your dashboard at `https://<your-travis-ci-enterprise-domain>:8800`.
+- Click on 'Settings' in the topmost menu.
+- Click 'Advanced Settings' in the left menu (or scroll down).
+- Copy and paste your certificate chain into the text box labeled 'Custom Certificate Authority (CA) Bundle'.
+- Click 'Save' at the bottom of the page.
+
+**Please note that this will require a restart of Travis CI Enterprise in order to take affect and may cause brief downtime.**
+
+Upon restart you can then verify whether SSL verification is now working. If you are still experiencing trouble even after setting the chain then please contact us at [enterprise@travis-ci.com](mailto:enterprise@travis-ci.com) for assistance.
+
+## Using a Let's Encrypt SSL Certificate
+
+Travis CI Enterprise works well together with a Let's Encrypt SSL Certificate. To obtain one for your domain, please follow the steps below.
+
+What you need:
+
+1. An email address Let's Encrypt can send emails to. This will be used to notify about urgent renewal and security notices.
+2. A domain name under which your installation is available (we're using `travis.example.com` in this guide).
+
+**Please note: This change will cause downtime for your system,for this reason, we recommend to perform this process in a maintenance window.**
+
+To obtain an SSL certificate we'll be using [certbot](https://certbot.eff.org/#ubuntutrusty-other). Certbot is available as an Ubuntu package.
+
+Please login to your platform machine via SSH and run the following steps to install certbot:
+
+```
+$ sudo apt-get update
+$ sudo apt-get install software-properties-common
+$ sudo add-apt-repository ppa:certbot/certbot
+$ sudo apt-get update
+$ sudo apt-get install certbot
+```
+
+`certbot` offers multiple ways to obtain a certificate. We'll pick the `temporary webserver` option since it doesn't require any additional configuration. The only prerequisite though is that the Travis CI Enterprise container has to be stopped so that webserver can bind to port 443 properly.
+
+***To start***, please stop Travis CI Enterprise:
```

Comment:
:-D

I'm kinda confused as to which numbered steps are which here. If you indent paragraphs to further then the base text, you can include them as part of the numbered list (also code blocks) 

1.  Like so
       
       ```
       Test
       ```
2. Test

Reply:
How about something like

1. Stop travis CI enterprise by running ...

2. Run the interactive certificate process by.... , selecting `X and then Y`

Reply:
I agree with this these suggestions but I would like to leave this as a followup. This is already present today, I simply copy/pasted it from the other page since we now have a central page for SSL-related material. I will add a TODO to my list to revisit this immediately following this PR.